src/admin/components/ApplicantStatus/ApplicantStatus.tsx

```diff
@@ -0,0 +1,34 @@
+import Badge from "react-bootstrap/Badge";
+
+import { Status } from "admin/utils/useApplicants";
+
+export const StatusLabels = {
+	[Status.accepted]: "accepted",
+	[Status.rejected]: "rejected",
+	[Status.waitlisted]: "waitlisted",
+	[Status.pending]: "needs review",
+	[Status.reviewed]: "reviewed",
```

Comment:
Clarification: In what case does status = reviewed but not accepted/rejected/waitlisted?

Reply:
I agree this isn't the cleanest, but I originally had it to allow reviewers to submit their reviews without affecting the actual status of an applicant which would be reflected in the portal (status would be reviewed until released). Maybe we could separate it into applicant status (pending review, decision released) and review status (accepted, rejected, waitlisted), but reviewers originally mentioned that we may need to account for multiple reviews submitted by one person.

Reply:
I see. It looks like applications have what they need so far, and the review process is not exactly flushed out yet, so no additional changes needed.